R/dups-class.R

```diff
@@ -0,0 +1,27 @@
+as.dups <- function(x) {
+  if (!is.data.frame(x) && !is(x, "dups")) {
+    stop("Can only convert 'data.frame' and 'dups' to 'dups'")
+  }
+  if (is(x, "dups")) {
+    return(x)
+  }
+  structure(
+    list(dups_df = x),
+    class = "dups"
+  )
+}
+
+as.data.frame.dups <- function(x) {
```

Comment:
this will be wrong if x$dups_df is a tibble; suggest changing to `as.data.frame(x$dups_df)`